ansible/library/port_alias.py

```diff
@@ -99,6 +101,9 @@ def get_portmap(self, asic_id=None, include_internal=False):
         # All asic names
         asic_if_names = []
 
+        #conn_graph = conn_graph_helper.get_conn_graph_facts([self.hostname])
+        conn_graph = {}
```

Comment:
Where is ```conn_graph``` initialized?

Reply:
Line 104 should be uncommented, line 105 should be removed. This was code that I used for testing the old behavior as well as testing error cases (in case `get_conn_graph_facts` didn't find any data for the port).